cirq-google/cirq_google/ops/dynamical_decoupling_tag.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import attrs
+
+from typing import Optional
+import cirq_google.api.v2.program_pb2 as v2
+
+
+SUPPORTED_DD_PROTOCOLS = frozenset(["X", "Y"])
```

Comment:
@eliottrosenberg which protocols should we support?

Reply:
I'll respond in a private message.